logger_test.go

```diff
@@ -245,3 +245,21 @@ func BenchmarkLogger_large(b *testing.B) {
 			WithError(err).Error("upload failed")
 	}
 }
+
+func TestLogger_logl(t *testing.T) {
+	h := memory.New()
+
+	l := &log.Logger{
+		Handler: h,
+		Level:   log.InfoLevel,
+	}
+
+	l.Debugl(func() string { assert.Fail(t, "Supplier should not be invoked."); return "ignored"})
```

Comment:
Tests that this supplier is never invoked otherwise the test would fail. If I remove line 181 with the level skip the test fails.